stream-transformation-test/sample-transformations/pom.xml

```diff
@@ -22,6 +24,14 @@
         <dependency>
             <groupId>uk.gov.justice.utils</groupId>
             <artifactId>test-utils-core</artifactId>
+            <version>${test.utils.core.version}</version>
```

Comment:
we can add these to the managed dependencies list in the repos top level pom, then we can exclude the version.